package.json

```diff
@@ -102,6 +102,7 @@
     "lodash.find": "^4.6.0",
     "lodash.includes": "^4.3.0",
     "node-fetch": "^1.6.3",
-    "parse-diff": "^0.4.0"
+    "parse-diff": "^0.4.0",
+    "voca": "^1.2.0"
```

Comment:
I come from the native world, where for libraries we try to use as few dependencies as possible unless its for something obviously outside of the context of danger (diff parsing for example)

But I guess I can adapt :D